packages/twenty-front/src/modules/object-record/record-board/components/RecordBoard.tsx

```diff
@@ -30,6 +30,9 @@ const StyledContainer = styled.div`
   flex: 1;
   flex-direction: row;
   min-height: calc(100% - 1px);
+  & > *:not(:first-child) {
+    border-left: 1px solid ${({ theme }) => theme.border.color.light};
+  }
```

Comment:
This change is here to replace the logic that was in place based on recoil state to determine which column if the first and witch is the last and apply or not the border.
This css is removing all the hardcoded logic as the boolean was not used anywhere else.